<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GammaRay::StyleElementStateTablePage</class>
 <widget class="QWidget" name="GammaRay::StyleElementStateTablePage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QTableView" name="tableView">
     <property name="selectionMode">
      <enum>QAbstractItemView::NoSelection</enum>
     </property>
     <property name="showGrid">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Cell &amp;Width:</string>
       </property>
       <property name="buddy">
        <cstring>widthBox</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="widthBox">
       <property name="suffix">
        <string>px</string>
       </property>
       <property name="maximum">
        <number>999</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Cell &amp;Height:</string>
       </property>
       <property name="buddy">
        <cstring>heightBox</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="heightBox">
       <property name="suffix">
        <string>px</string>
       </property>
       <property name="maximum">
        <number>999</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>&amp;Zoom:</string>
       </property>
       <property name="buddy">
        <cstring>zoomSlider</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="zoomSlider">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>8</number>
       </property>
       <property name="pageStep">
        <number>1</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
